### PR TITLE
Remove obsolete files

### PR DESCRIPTION
**Pull request by badfiles**

### CODE REVIEW

**Comment by badfiles** [comment]
The code is no longer used, so I would rather ask why it was left. 
Having unused code is not a bug, maybe you've forgotten to remove it, or maybe you have plans for it.

File: bug_file_add.php [hunk 1]
```diff
@@ -1,90 +0,0 @@
-<?php
```
Comment (k1931547) [inline bug_file_add.php]: commits removes all file

File: bug_file_upload_inc.php [hunk 2]
```diff
@@ -1,113 +0,0 @@
-<?php
```
Comment (k1931547) [inline bug_file_upload_inc.php]: commit removes all file

**Comment by k2152262** [comment]
@vboctor @giallu @daryn @rombert @atrol @grangeway @dregad @dentxinho @bieli @tomcarlson @fengkuok @syncguru @diegovice @caribe @fager @pikiel @nikoutel @6run0 @AgnidykhBayky @sayyidsh @pixelstreamAu @johnms @mikeching @g-taki @benv8nam @stew721 @sayazyi @mamoniem @lezouretalice @suriyaakudolsc @solovets @kacmem @xiaojiangang @chenyongze @myu9882 @markmarco16 @mgretler @lincolnwung @zedxxx @krime @scottmatheina @faviogalvis @vand3rson @like9729 @badfiles @zenithtekla @sureshinde @peterdavehello @netsf @csatwood @miao606 @bitlin @ioroid @supertassu @ulkuderner @vbathke @hersonpc @jrtorres42 @thejuanwhocode @cyuankai @mantianxingxing @boby75 @lordshell @burningwind @nospam1961 @planetbrett @gantha @pbcreativesolutions @bastilimbach @ltpgt @modir @dharmeshkakadia @xasos @LewisJEllis progress on implementing feedback ?

**Comment by sync1** [comment]
@vboctor @giallu @daryn @rombert @atrol @grangeway @dregad @Dentxinho @bieli @tomcarlson @syncguru @diegovice @caribe @fager @nikoutel @6RUN0 @AgnidykhBayky @sayyidsh @PixelstreamAU @JohnMS @mikeching @g-taki @benv8nam @stew721 @sayazyi @lezouretalice @SuriyaaKudoIsc @solovets @kacmem @xiaojiangang @chenyongze @myu9882 @markmarco16 @mgretler @lincolnwung @krime @scottmatheina @FavioGalvis @vand3rson @like9729 @badfiles @zenithtekla @SureShinde @netsf @csatwood @miao606 @mdj20130907 @bitlin @ioroid @supertassu @TasCraft @fernandodelucca @ulkuderner @vbathke @hersonpc @jrtorres42 @thejuanwhocode @cyuankai @mantianxingxing @shirleycharlin @boby75 @Lordshell @BurningWind @jaguarxii @Praymundo @johnzastrow @ewmartinez @marcioklein @lypzln @nospam1961 @jllano @lincoln56 @modir @xiaocnie @sivpro @planetbrett @teodortalov @ltpgt @pbcreativesolutions @Gantha @Gerolkae @gunsmoker @nikerabbit @translatewiki  whats status ?

**Pull request merged**